lib/action/init.ml

```diff
@@ -7,8 +7,12 @@ let run (module R : Sigs.RESOLVER) =
       ~on:`Source R.Source.chain
   in
   let+ cache, members =
+    let where path =
+      Yocaml.Path.has_extension "yml" path ||
+      Yocaml.Path.has_extension "yaml" path
+    in
     Yocaml.Action.fold ~only:`Files
-      ~where:(Yocaml.Path.has_extension "yml")
```

Comment:
Instead of removing the filter, I suggest to make it allowing `yml` and `yaml` ? I Can Do it on your branch if you want?

Reply:
That's what I did - please see the definition of `where` a few lines above :-)